src/tasks/component-installers/cert-manager.ts

```diff
@@ -129,11 +133,22 @@ export class CertManagerTasks {
       },
       {
         title: 'Set up Eclipse Che certificates issuer',
-        task: async (_ctx: any, task: any) => {
-          const cheClusterIssuerExists = await this.kubeHelper.clusterIssuerExists('che-cluster-issuer')
+        task: async (ctx: any, task: any) => {
+          const clusterIssuers = await this.kubeHelper.listClusterIssuers(CHE_RELATED_COMPONENT_LABEL, ctx.certManagerK8sApiVersion)
```

Comment:
If there is the only one clusterIssuer without the label then we have to use it.
Do we handle that?

Reply:
Currently no. The idea was to use the label for such purpose.
It makes sense, though, will add such functionality.